docs/how-to/build-generator-to-log-pipeline.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: "How to build a generator-to-log pipeline"
+sidebar_position: 0
+---
+
+In this guide, we'll learn how to build a basic Conduit pipeline. It will use
+two built-in connectors:
+
+- A generator source (a source connector that generates random data). Our
+  pipeline will generate imaginary employee data.
+- A logging destination (a destination that simply logs all incoming records).
+
+As you'll see, all it takes to accomplish this is to write
+a [pipeline configuration file](/docs/pipeline-configuration-files/getting-started)!
+
+The steps below will guide you through installing Conduit and gradually building
+the pipeline. If you're in a hurry, you can skip ahead
+and [run the pipeline](#step-6-run-the-pipeline) immediately.
+
+## Step 1: Install Conduit
+
+Download
+the [latest Conduit release](https://github.com/ConduitIO/conduit/releases/latest)
+to a directory of your choice. In this guide, the directory will be
+`~/conduit-playground`.
+
+## Step 2: Create a pipeline configuration file
+
+Pipelines can be created through YAML configuration files, that are, by default,
+put into a directory called `pipelines`.
+
+With the following command, we'll create the `pipelines` directory and a file
+that will contain our pipeline's configuration:
+
+```bash
+mkdir pipelines && touch pipelines/generator-to-log.yaml
+```
+The directory layout should be as below:
+
+```console
+~/conduit-playground
+├── conduit
+└── pipelines
+    └── generator-to-log.yaml
+```
+
+
+## Step 3: The pipeline's basics
+
+Next, write the following to `generator-to-log.yaml`:
+
+```yaml
+version: "2.2"
+pipelines:
+  - id: pipeline1
+    # Tells Conduit to run the pipeline automatically
+    status: running
+    name: pipeline1
+    description: A generator-to-log pipeline
+```
+
+## Step 4: Add the generator source
+
+Now we can start adding connectors to the pipeline. add a source connector under `connectors` in the configuration file:
+
+```yaml
+    connectors:
+      - id: source1
+        type: source
+        plugin: builtin:generator
+        settings:
+          format.type: structured
+          format.options.id: int
+          format.options.name: string
+          rate: "2s"
+```
+
+The fields `id`, `type`, `plugin` and `settings` are found in every connector.
+Every connector has its own set of configuration parameters. The generator's
+configuration can be
+found [here](https://github.com/ConduitIO/conduit-connector-generator?tab=readme-ov-file#configuration).
+
+The configuration below instructs the generator to produce 1 record with a
```

Comment:
```suggestion
The configuration above instructs the generator to produce 1 record every 2 seconds with a
```

Reply:
Good catch! I fixed that literally a minute ago.:D